Dockerfile

```diff
@@ -0,0 +1,48 @@
+FROM debian:buster-slim
+
+RUN set -x \
+  `# workaround for error in debian-slim during openjdk installation` \
+  && mkdir -p /usr/share/man/man1 \
+  && apt-get update \
+  && apt-get install --no-install-recommends -y curl git gnupg openjdk-11-jre-headless python3 python3-pip \
```

Comment:
While you're here: maybe move `python-pip3` to where `build-essential python3-dev` are installed; this way the first three layers can be shared b/w conda and pip images (without installing `pip` in conda image)